Hunting Queries/SecurityEvent/hunt_LOLBins.yaml

```diff
@@ -0,0 +1,28 @@
+id: 4a6030de-c11c-4206-b4ea-cd005ebf80a2
+name: LOLBins and LOLScripts
+description: |
+  'This query identifies Microsoft-signed Binaries and Scripts that are not system initiated. This technique is commonly used in phishing attacks'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Execution
+relevantTechniques:
+  - T1218
+  
+query: |
+  //This query uses sysmon data depending on table name used this may need updataing
+  let Bin = externaldata(Binary: string) [@"https://raw.githubusercontent.com/sonnyakhere/LOLBAS_to_CSV/main/lolbas.csv"] with (format="csv", ignoreFirstRecord=True);
+  let ioc = dynamic(["http", "ftp"]);
+  SecurityEvent
+  | where TimeGenerated between ( datetime(2022-08-07, 00:00) .. datetime(2022-08-07, 23:59) )
```

Comment:
Sorry for the delay,  few more changes that i missed in first review.  
 - Please remove hardcoded datetimes from the query and replace with now() or ago(1d) etc. 
 - Can you add filter for eventId 4688 as these are process execution events. 
 - Chage the name be more descriptive eg. Suspicious command line tokens in LolBins or LolScripts etc. 
 - remove the comment about sysmon data as current query is standard 4688. You can add Sysmon events too in the query and then include that note.